README.md

```diff
@@ -361,6 +363,26 @@ $ trivy repo https://github.com/knqyf263/trivy-ci-test
 
 Only public repositories are supported.
 
+## Podman
+[EXPERIMENTAL] This feature might change without preserving backwards compatibility.
+
+Scan your image in Podman (>=2.0) running locally. The remote Podman is not supported.
+Before performing Trivy commands, you must enable the podman.sock systemd service on your machine.
+For more details, see [here](https://github.com/containers/podman/blob/master/docs/tutorials/remote_client.md#enable-the-podman-service-on-the-server-machine)
+
+```
+$ systemctl --user enable --now podman.socket
+```
+
+Then, you can scan your image in Podman.
+
+```
+$ podman images
+REPOSITORY                TAG     IMAGE ID      CREATED      SIZE
+localhost/test            latest  ed1b0762d815  3 weeks ago  5.85 MB
```

Comment:
Maybe we could explain how we get that localhost/test image?

When I was testing I run:

```console
$ podman pull docker.io/library/alpine:3.10.2
$ podman tag docker.io/library/alpine:3.10.2 localhost/alpine:3.10.2

$ trivy image localhost/alpine:3.10.2
```

Reply:
Thanks. I've done. https://github.com/aquasecurity/trivy/pull/825/commits/8c241934dd36c2022d1c8d794e37e7087ef69c55